addons/modules/stringtable.xml

```diff
@@ -857,6 +857,16 @@
             <Japanese>オブジェクトの高さをこの値で変更します (負の値も可能).</Japanese>
             <Korean>이 값에 따라 물체의 고도를 변경합니다. (음수도 가능)</Korean>
         </Key>
+        <Key ID="STR_ZEN_Modules_ScaleObject">
+            <English>Scale Object</English>
+            <French>Redimensionner Objet</French>
+            <German>Objekt skalieren</German>
+        </Key>
+        <Key ID="STR_ZEN_Modules_ScaleObject_Tooltip">
+            <English>Scale Object by this factor.</English>
```

Comment:
Lowercase 'o' in object. Also, "**by** this factor" kind of implies that the new scale is affected by the current scale. Maybe change to "Scale the object by this factor relative to the original model."